cartography/graph/cleanupbuilder.py

```diff
@@ -104,13 +104,33 @@ def _build_cleanup_node_and_rel_queries(
         )
 
     # Ensure the node is attached to the sub resource and delete the node
-    query_template = Template(
-        """
-        MATCH (n:$node_label)$sub_resource_link(:$sub_resource_label{$match_sub_res_clause})
-        $selected_rel_clause
-        $delete_action_clause
-        """,
-    )
+    if node_schema.sub_resource_relationship.target_node_label == "AWSAccount":
+        query_template = Template(
+            """
+            MATCH (n:$node_label)$sub_resource_link(:$sub_resource_label{$match_sub_res_clause})
+            <-[:OWNER]-(:AWSOrganization{id: $ORGANIZATION_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})
+            $selected_rel_clause
+            $delete_action_clause
+            """,
+        )
+    elif node_schema.sub_resource_relationship.target_node_label == "GitHubOrganization":
+        query_template = Template(
+            """
+            MATCH (n:$node_label)$sub_resource_link(:$sub_resource_label{$match_sub_res_clause})
+            <-[:OWNER]-(:GitHubOrganization{id: $ORGANIZATION_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})
```

Comment:
`GitHubOrganization` is sub_resource_label and it already in model, we need to add only 
`<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})` here

Reply:
Done